docs/en/cloud/marketplace.md

```diff
@@ -33,37 +34,39 @@ title: CSP Marketplace
 
 
-### Azure is coming soon
+### Azure (coming soon)
+
+## FAQs
 
-## How can I understand that my organization is connected to the marketplace billing?
+### How can I verify if my organization is connected to CSP marketplace billing?
 
 In ClickHouse Cloud console, navigate to **Admin** -> **Billing**. You should see the name of the marketplace and the link in the **Payment details section**
 
-## I am an existing ClickHouse Cloud user. What will happen if I subscribe to the CH Cloud via aws marketplace?
+### I am an existing ClickHouse Cloud user. What will happen if I subscribe to ClickHouse Cloud via AWS marketplace?
 
 A separate organization connected to the marketplace will be created. Your existing services and organizations will remain and they will not be connected to the marketplace billing.
 
 You can switch between organizations in the top right corner of the ClickHouse Cloud console.
 
-## I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
+### I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
 
-Please contact [ClickHouse Cloud support](https://clickhouse.cloud/support) in this case.
+Please follow the steps outlined above to subscribe and sign up via the marketplace. A new organization will be created for you. After this, contact [ClickHouse Cloud support](https://clickhouse.cloud/support) and share the organization ID for the old organization (the one which has existing services and workloads), and the new org (that you created via the marketplace). You will also need to share the email for a user that has the admin role on both orgs.  ClickHouse Cloud support team will then be able to connect your existing services to the new org so that your usage starts getting billed via the marketplace.
 
-## I subscribed as a marketplace user and now I want to unsubscribe from the ClickHouse Cloud.
+### I subscribed as a marketplace user and now I want to unsubscribe from ClickHouse Cloud.
 
 Note that you can simply stop using ClickHouse Cloud and delete all existing ClickHouse Cloud services. Even though the subscription will still be active, you will not be paying anything as ClickHouse Cloud doesn't have any recurring fees.
 
-If you want to unsubscribe, please navigate to the Cloud Provider console and cancel the subscription renewal there. Once the subscription ends, all existing services will be stopped and you will be prompted to add a credit card. If no card was added, after two weeks all existing services will be deleted.
+If you want to unsubscribe, please navigate to the Cloud Provider console and cancel the subscription renewal there. Once the subscription ends, all existing services will be stopped and you will be prompted to add a credit card. If no card is added, after two weeks all existing services will be deleted.
```

Comment:
We've matched this flow with the credit card flow:
Cancelling the MP subscription will start the clock on the following actions:
* Once we get the subscription cancellation notice from Tackle the org can no longer create new services or start stopped services
* 14 days after the cancellation notice from tackle we stop the services (they can't be started by the user)
* 30 days after the cancellation notice from tackle we decommission the organization, which will include deleting all services and their data.